components/Image/image.html

```diff
@@ -1,20 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/829cb68a23.js" crossorigin="anonymous"></script>
```

Comment:
change this to `<script defer src="https://kit.fontawesome.com/829cb68a23.js" crossorigin="anonymous"></script>` this will load the js afterwards and won't cause problems